pkg/webui/console/store/middleware/logics/devices.js

```diff
@@ -137,6 +138,38 @@ const getDevicesListLogic = createRequestLogic({
   },
 })
 
+const getDevicesPeriodicallyLogic = createLogic({
```

Comment:
Please see my other comment on removing this periodic fetch for the time being (and just using the normal get logic instead).